backend/src/database/queries.js

```diff
@@ -12,6 +12,9 @@ const CREATE_TASKS_TABLE_QUERY = `CREATE TABLE IF NOT EXISTS tasks (
   isDone INTEGER,
   text TEXT,
   users TEXT,
+  author TEXT DEFAULT '',
+  priority TEXT DEFAULT 'Low',
+  status TEXT DEFAULT 'Planned',

```

Comment:
@eugeek btw, please discuss this with Ksusha, we won't have `Planned` as default in the latest design.